packages/vaadin-overlay/src/vaadin-overlay-position-mixin.js

```diff
@@ -67,125 +77,131 @@ export const _PositionMixin = (superClass) =>
     }
     static get observers() {
       return [
-        `__positionSettingsChanged(positionTarget, horizontalAlign, verticalAlign,
-      noHorizontalOverlap, noVerticalOverlap)`
+        `__positionSettingsChanged(positionTarget, horizontalAlign, verticalAlign, noHorizontalOverlap, noVerticalOverlap)`,
+        `__overlayOpenedChanged(opened)`
```

Comment:
nit: we probably don't need backticks here

```suggestion
        '__positionSettingsChanged(positionTarget, horizontalAlign, verticalAlign, noHorizontalOverlap, noVerticalOverlap)',
        '__overlayOpenedChanged(opened)'
```